tests/Unit/Impl/StreamxClientIntegrationTest.php

```diff
@@ -40,21 +43,21 @@ class StreamxClientIntegrationTest extends TestCase {
 
     public static function setUpBeforeClass(): void {
         self::$client = StreamxClientBuilders::create(self::INGESTION_BASE_URL)->build();
-        self::$publisher = self::$client->newPublisher("pages", self::PAGES_SCHEMA_NAME);
+        self::$publisher = self::$client->newPublisher(self::PAGES_CHANNEL, self::PAGE_SCHEMA_NAME);
         self::$page = new Page(new Content(self::CONTENT));
         self::$pageArray = ['content' => ['bytes' => self::CONTENT]];
     }
 
-    //** @test */
+    /** @test */
     public function shouldPublishAndUnpublishPageObject() {
         $this->shouldPublishAndUnpublishPagePayload(
             self::PAGE_OBJECT_KEY,
             self::$page
         );
     }
 
-    //** @test */
-    public function shouldPublishAndUnpublishPageArray() {
+    /** @test */
+    public function shouldPublishAndUnpublishPagArray() {
```

Comment:
you added typo in name "PagArray"

Reply:
Yes, thanks for spotting that, I'll also restore disabiling the tests (since they need a running StreamX local instance)